packages/react/src/useUploadThing.ts

```diff
@@ -14,6 +14,7 @@ import type {
   inferEndpointOutput,
   inferErrorShape,
 } from "uploadthing/server";
+import type { MaybePromise } from "../../uploadthing/src/internal/types";
```

Comment:
This import path wont work. I dont know if we export it from a public entrypoint, if not you can just inline the union type i guess

Reply:
@juliusmarminge updated